react/Autosuggest/Autosuggest.js

```diff
@@ -51,21 +50,23 @@ export default class Autosuggest extends Component {
     showMobileBackdrop: false
   };
 
-  constructor() {
-    super();
+  state = {
+    areSuggestionsShown: false
+  }
 
-    this.storeInputReference = this.storeInputReference.bind(this);
-    this.storeTextFieldReference = this.storeTextFieldReference.bind(this);
-    this.renderInputComponent = this.renderInputComponent.bind(this);
+  componentWillUpdate(nextProps, nextState) {
```

Comment:
🤔  Seems you're right. Was thinking this component owned the the suggestions container for some reason.